pkg/gatherers/conditional/conditional_gatherer.go

```diff
@@ -88,18 +41,29 @@ type Gatherer struct {
 	metricsGatherKubeConfig *rest.Config
 	imageKubeConfig         *rest.Config
 	firingAlerts            map[string]bool // golang doesn't have sets :(
+	gatheringRulesEndpoint  string
+	gatheringRules          []GatheringRule
+	httpClient              *http.Client
 }
 
 // New creates a new instance of conditional gatherer with the appropriate configs
-func New(gatherProtoKubeConfig, metricsGatherKubeConfig *rest.Config) *Gatherer {
-	imageKubeConfig := rest.CopyConfig(gatherProtoKubeConfig)
-	imageKubeConfig.QPS = common.ImageConfigQPS
-	imageKubeConfig.Burst = common.ImageConfigBurst
+func New(gatherProtoKubeConfig, metricsGatherKubeConfig *rest.Config, gatheringRulesEndpoint string) *Gatherer {
+	var imageKubeConfig *rest.Config
+	if gatherProtoKubeConfig != nil {
+		imageKubeConfig = rest.CopyConfig(gatherProtoKubeConfig)
+		imageKubeConfig.QPS = common.ImageConfigQPS
+		imageKubeConfig.Burst = common.ImageConfigBurst
```

Comment:
Just out of curiosity - why do you set these?

Reply:
If I remember correctly, getting image streams didn't work without these

Reply:
OK thanks. Maybe we could mention it in a comment.